src/xdocs/checks/javadoc/atclauseorder.xml

```diff
@@ -33,50 +33,41 @@
               <td>6.0</td>
             </tr>
             <tr>
               <td>target</td>
               <td>Specify block tags targeted.</td>
-              <td>subset of tokens
-                <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html">
-                  TokenTypes
-                </a>
-              </td>
+              <td>subset of tokens <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html">TokenTypes</a></td>
               <td>
                 <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CLASS_DEF">
-                  CLASS_DEF</a>
+                    CLASS_DEF</a>
                 , <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#COMPACT_CTOR_DEF">
-                  COMPACT_CTOR_DEF</a>
+                    COMPACT_CTOR_DEF</a>
                 , <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CTOR_DEF">
-                  CTOR_DEF</a>
+                    CTOR_DEF</a>
                 , <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ENUM_DEF">
-                  ENUM_DEF</a>
+                    ENUM_DEF</a>
                 , <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#INTERFACE_DEF">
-                  INTERFACE_DEF</a>
+                    INTERFACE_DEF</a>
                 , <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#METHOD_DEF">
-                  METHOD_DEF</a>
+                    METHOD_DEF</a>
                 , <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#RECORD_DEF">
-                  RECORD_DEF</a>
+                    RECORD_DEF</a>
                 , <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#VARIABLE_DEF">
-                  VARIABLE_DEF</a>
+                    VARIABLE_DEF</a>
+                  .
               </td>
               <td>6.0</td>
             </tr>
             <tr>
               <td>violateExecutionOnNonTightHtml</td>
-              <td>
-                Control when to print violations if the Javadoc being examined by this check
-                violates the tight html rules defined at
-                <a href="../../writingjavadocchecks.html#Tight-HTML_rules">Tight-HTML Rules</a>.
-              </td>
-              <td><a href="../../property_types.html#boolean">boolean</a>
-              </td>
+              <td>Control when to print violations if the Javadoc being examined by this check violates the tight html rules defined at <a href="../../writingjavadocchecks.html#Tight-HTML_rules">
+     Tight-HTML Rules</a>.</td>
+              <td><a href="../../property_types.html#boolean">boolean</a></td>
               <td><code>false</code></td>
-              <td>8.3</td>
+              <td>6.0</td>
```

Comment:
I think the change in version number might be related to https://github.com/checkstyle/checkstyle/issues/13934.

Reply:
This version update is significant, we can not doing it.
Let's keep this check in array of macros but no migration for now.

Reply:
done

Reply:
@Romani, was any PR merged which changed the version to 6.0? Because, i see in the master it to be 6.0. And, hence, it doesn't actually is registered as a change in my push which was the case earlier.

Reply:
> @romani, was any PR merged which changed the version to 6.0? Because, i see in the master it to be 6.0. And, hence, it doesn't actually is registered as a change in my push which was the case earlier.

What is the actual version? 6.0 or 8.3?

Reply:
> @romani, was any PR merged which changed the version to 6.0? Because, i see in the master it to be 6.0. And, hence, it doesn't actually is registered as a change in my push which was the case earlier.

My bad. I assume the earlier push changed the version of `8.3` of property `violateExecutionOnNonTightHtml` to 6.0, which is not thankfully happening in the recent push.